bitnami/odoo/templates/ingress.yaml

```diff
@@ -42,6 +42,11 @@ spec:
             pathType: {{ default "ImplementationSpecific" .pathType }}
             {{- end }}
             backend: {{- include "common.ingress.backend" (dict "serviceName" (include "common.names.fullname" $) "servicePort" "http" "context" $) | nindent 14 }}
+          - path: /longpolling/
+            {{- if eq "true" (include "common.ingress.supportsPathType" .) }}
+            pathType: {{ .Values.ingress.pathType }}
+            {{- end }}
+            backend: {{- include "common.ingress.backend" (dict "serviceName" (include "common.names.fullname" .) "servicePort" "longpolling" "context" $)  | nindent 14 }}
```

Comment:
This is being added inside a range for extraHosts, so it won't work as you expect. I think this `/longpolling/` path should be added to the previous host section.